lxqt-config/lxqt-config-appearance/lxqt-config-appearance_ca.ts

```diff
@@ -0,0 +1,225 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="ca">
+<context>
+    <name>FontsConfig</name>
+    <message>
+        <location filename="../../../fontsconfig.ui" line="23"/>
+        <source>Font</source>
+        <translation>Tipus de lletra</translation>
+    </message>
+    <message>
+        <location filename="../../../fontsconfig.ui" line="30"/>
+        <source>Default font for user interface</source>
+        <translation>Tipus de lletra predeterminat per a la interfície d&apos;usuari</translation>
+    </message>
+    <message>
+        <location filename="../../../fontsconfig.ui" line="36"/>
+        <source>Font name:</source>
+        <translation>Nom del tipus de lletra:</translation>
+    </message>
+    <message>
+        <location filename="../../../fontsconfig.ui" line="46"/>
+        <source>Style:</source>
+        <translation>Estil:</translation>
+    </message>
+    <message>
+        <location filename="../../../fontsconfig.ui" line="53"/>
+        <source>Point size:</source>
+        <translation>Mida del punt:</translation>
+    </message>
+    <message>
+        <location filename="../../../fontsconfig.ui" line="68"/>
+        <source>Normal</source>
+        <translation>Normal</translation>
+    </message>
+    <message>
+        <location filename="../../../fontsconfig.ui" line="73"/>
+        <source>Bold</source>
+        <translation>Negreta</translation>
+    </message>
+    <message>
+        <location filename="../../../fontsconfig.ui" line="78"/>
+        <source>Italic</source>
+        <translation>Cursiva</translation>
+    </message>
+    <message>
+        <location filename="../../../fontsconfig.ui" line="83"/>
+        <source>Bold Italic</source>
+        <translation>Cursiva en negreta</translation>
+    </message>
+    <message>
+        <location filename="../../../fontsconfig.ui" line="94"/>
+        <source>The following settings only affect newly started applications</source>
+        <translation>Els següents ajusts únicament tindran efecte amb les aplicacions iniciades a partir d&apos;ara</translation>
```

Comment:
Are you adding the accelerator shortcuts based on your private preferences (when there is no accelerator in original)?